blogposts/2023-04-27-announcing-macos-71.md

```diff
@@ -0,0 +1,55 @@
+---
+post_title: "Introducing React Native macOS 0.71"
+author1: sanajmi
+post_slug: 2023-04-27-announcing-macos-71
+post_date: 2023-04-27 12:00:00
+categories: react-native
+tags: react-native, announcement, launch, react-native-macos, release
+summary: "A new phase for react-native-macos: introducing version 0.71 and experimental support for the new architecture!"
+---
+
+Hello everyone,
+
+We're thrilled to announce the latest release of React Native macOS: 0.71. There’s a lot to talk about with this new minor version, so let’s get right into it!
+
+### Catching up to core
+
+This is the first time that react-native-macos is caught up with its [iOS, Android](https://github.com/facebook/react-native) and [Windows](https://github.com/microsoft/react-native-windows) counterparts, and our goal is to keep it that way going forward. We have already started working on 0.72 to ensure that developers targeting all the platforms can swiftly upgrade across the board once the new version comes out.
+
+To reach this goal, as some of you might have noticed, we had to completely skip some minor releases (such as 0.69 and 0.70). We had to take this tradeoff to accommodate our internal challenges, motivated by the fact that it is the same code that we use in our products (very much like how react-native core is used by Meta). For this reason, it will get more stable as we use it more internally: we push a new patch release on every commit so you can expect in the future new patches to be published as we quickly catch and fix bugs.
+
+### A successful collaboration
+
+A lot of work went into making React Native macOS better for this release, and a special thanks must be given to the Meta and Messenger Desktop engineers that regularly submitted fixes and improvements. In particular, we wanted to shout out [Shawn Dempsey](https://github.com/shwanton) and [Christoph Purrer](https://github.com/christophpurrer) for their great work on the many component (such as TextInput and ScrollView) and adding support for Fabric, among many other things!
+
+### Bringing the new architecture from iOS to macOS
+
+Talking about Fabric: the reason why react-native-macos is a fork and not a fully independent implementation, is that a lot of the existing iOS code can be easily massaged into working for macOS. While this introduces extra complexity in its regular maintenance, it allows to quickly get access to new features -- including the new architecture.
+
+With this release, we are excited to introduce an experimental preview of Fabric for macOS!
```

Comment:
note from @stmoy: what are the limitations? Worth adding a mention?

Reply:
There are quite a few limitations. Stuff doesn't render correctly everywhere and I think the app hangs if you click a button 11 times. It is very much not usable, but more a proof of concept.

Reply:
added an extra line to clarify that it's in proof of concept stage